salt/state.py

```diff
@@ -3742,7 +3742,8 @@ def __gen_opts(self, opts):
             )
             opts["env_order"] = mopts.get("env_order", opts.get("env_order", []))
             opts["default_top"] = mopts.get("default_top", opts.get("default_top"))
-            opts["state_events"] = mopts.get("state_events")
+            if "state_events" not in opts:
```

Comment:
**nitpick (non-blocking):** this could be an `or` like the line below

Reply:
Good point - updated that so it's consistent.